### PR TITLE
Update intro paragraph

### PR DESCRIPTION
**Pull request by lksbrssr**
I've tried to add the retrieval element here (question: Okay to include this even if retrieval markets are still somewhat WIP?) and I've tried to include the idea that users (questions: What do we call the agents that act on the Filecoin storage network? Is 'users' correct?) can decide how long their files should be available (incl. forever). I think this latter point is important in our positioning vs. other web3 storage solutions.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ meta:
 
 # Filecoin Documentation
 
-**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+**Filecoin is a peer-to-peer network that allows anyone to store and retrieve files on the internet. Built-in economic incentives ensure that files are stored and retrieved reliably and continuously for however long a user specifies.**
 
-Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+Available storage capacity and the pricing for storing and retrieving files are not controlled by any single company. Instead, the Filecoin storage network facilitates open markets for storing and retrieving files that anyone can participate in. It uses its native cryptocurrency (FIL) to faciliate these transactions.
```
Comment (johnnymatthews) [inline docs/README.md:15]:
```suggestion
No single company or organization controls available storage capacity and the pricing for storing and retrieving files. Instead, the Filecoin storage network facilitates open markets for storing and retrieving files that anyone with access to the internet can participate in. It uses its native cryptocurrency (FIL) to facilitate these transactions.
```

File: docs/README.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ meta:
 
 # Filecoin Documentation
 
-**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+**Filecoin is a peer-to-peer network that allows anyone to store and retrieve files on the internet. Built-in economic incentives ensure that files are stored and retrieved reliably and continuously for however long a user specifies.**
```
Comment (johnnymatthews) [inline docs/README.md:13]:
```suggestion
Filecoin is a peer-to-peer network that allows anyone to store and retrieve data on the internet. Built-in economic incentives ensure that files are stored and retrieved reliably and continuously for however long a user specifies.

```

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
> question: Okay to include this even if retrieval markets are still somewhat WIP?

No, we should leave this out for now until we've got something more solid to work with.

> What do we call the agents that act on the Filecoin storage network? Is 'users' correct?

Users is correct.

> I think this latter point is important in our positioning vs. other web3 storage solutions.

Agreed, let's keep that in there.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**